src/ResourceManager/Profile/Commands.Profile.Test/EnvironmentCmdletTests.cs

```diff
@@ -105,6 +105,7 @@ public void AddsAzureEnvironmentUsingARMEndpoint()
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ResourceManager], cmdlet.ResourceManagerEndpoint);
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ActiveDirectory], cmdlet.ActiveDirectoryEndpoint);
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ActiveDirectoryServiceEndpointResourceId], cmdlet.ActiveDirectoryServiceEndpointResourceId);
+            Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.GraphEndpointResourceId], cmdlet.GraphAudience);
```

Comment:
`cmdlet.GraphAudience `is set from `RetrieveMetaDataEndpoints` function. Otherwise it will be null. We only have `ArmEndpoint` parameter in `Add-AzureRmEnvironment `cmdlet in this test call.